File: dotnet/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/Models/Concrete/WindowsButton.cs
using AbstractFactoryDesignPattern.Example1.Models.Abstract;

namespace AbstractFactoryDesignPattern.Example1.Models.Concrete;

public class WindowsButton : Button
{
	public override void Render()
	{
		Console.WriteLine("Rendering Windows Button");
	}
	public override void OnClick()
	{
		Console.WriteLine("Windows Button Clicked");
	}
}

File: java/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/src/main/java/creational/abstractfactory/example1/models/concretes/WindowsButton.java
package creational.abstractfactory.example1.models.concretes;

import creational.abstractfactory.example1.models.abstracts.Button;

public class WindowsButton extends Button {
    public void render()   { System.out.println("Rendering Windows Button"); }
    public void onClick()  { System.out.println("Windows Button Clicked"); }
}
